halotools/mock_observables/pair_counters/marked_npairs_3d.py

```diff
@@ -66,14 +76,6 @@ def marked_npairs_3d(sample1, sample2, rbins,
         containing the weights used for the weighted pair counts. If this parameter is
         None, the weights are set to np.ones(*(N1,N_weights)*).
 
-    weight_func_id : int, optional
-        weighting function integer ID. Each weighting function requires a specific
-        number of weights per point, *N_weights*.  See the Notes for a description of
-        available weighting functions.
-
-    verbose : Boolean, optional
```

Comment:
There are a lot of functions that take a `verbose` argument but never use it. I've removed it from most of those.